beer-garden/react-ui
# 167 - Long text in table shows ellipsis

* Updated the Table Component with an OverflowCellRenderer as the Default Cell - this will replace long text breaks with an ellipsis and a tooltip with the full text
* To use a normal cell, like the Explore Button on the Systems Index page, you can pass in the Cell:DefaultCellRenderer property in the column spec (see src/pages/SystemIndex/systemIndexHelpers for example)  

To test:
* Resize pages on the Systems Index page, should be ellipsis for overflow cells. Hover over for tooltip with full text. Explore button column should remain the same and not have an overflow tooltip.
